ikasaneip/developer/mvn-archetype/ikasan-standalone-filesystem-im/src/main/resources/archetype-resources/jar/src/test/java/com/ikasan/sample/spring/boot/builderpattern/ApplicationTest.java

```diff
@@ -115,27 +122,35 @@
     @DirtiesContext
     public void sourceFileFlow_flow() throws Exception
     {
+        flowTestRule.withFlow((Flow) moduleUnderTest.getFlow("${sourceFlowName}"));
```

Comment:
Change the variable declaration of
`Module moduleUnderTest;`
to 
`Module<Flow> moduleUnderTest;`
Then you dont have to caste in the code.